Assets/Unit Tests/Variables/ClampVariables.cs

```diff
@@ -0,0 +1,250 @@
+﻿using NUnit.Framework;
+using UnityEngine;
+// ReSharper disable CheckNamespace
+
+namespace ScriptableObjectArchitecture.Tests
+{
+    [TestFixture]
+    internal sealed class ClampVariables
+    {
+        [Test]
+        public void AssertThatFloatVariablesClampValues()
+        {
+            var variable = ScriptableObject.CreateInstance<FloatVariable>();
+            variable.IsClamped = true;
+            variable.MinClampValue = 0f;
+            variable.MaxClampValue = 5f;

```

Comment:
The values used for min and max here would make nice constants.  I mean, not that we'd really go and change them, but it'd be that much easier to read.

Reply:
Sure, I can make these consts

Reply:
Fixed this for all tests.